community/onboarding-offboarding.md

```diff
@@ -5,50 +5,51 @@ toc=true
 # Onboarding/Offboarding
 
 ## How to onboard someone in KOSS?
-
 1. Add them to the “Newbies” GitHub team.
 1. Add them to the kossiitkgp@ Google Group. Check the delivery settings. Anyone can ask to join using this link https://groups.google.com/g/kossiitkgp
-1. Add them to the Slack.
+1. Add them to the Slack. Ask each one to install the Slack desktop and mobile apps and enable notifications to not miss out any important updates.
 1. Add them to their batch’s private channel.
-1. Add their contact details on https://github.com/kossiitkgp/secrets .
+1. Add their contact details on https://github.com/kossiitkgp/secrets.
 1. Assign their 1:1 group.
 1. Add their name to the website.
```

Comment:
Should we also add bhattu here or are we planning on some automation to do that?

Reply:
Adding for now.